core/src/test/java/org/apache/iceberg/hadoop/TestTableSerialization.java

```diff
@@ -140,4 +194,21 @@ private static Table getMetaDataTable(Table table, MetadataTableType type) {
       throw new RuntimeException("Could not read object ", e);
     }
   }
+
+  private void checkTable(Table expected, Table actual, boolean completeCheck) {
```

Comment:
I would just do two checkTableFunctions rather than the boolean behavior switch but that's just me :)

Reply:
The boolean flag isn't descriptive. I'll switch to two methods instead.

Reply:
Fixed.